rollup.config.js

```diff
@@ -7,7 +7,7 @@ import autoPreprocess from 'svelte-preprocess'
 export default {
   input: "src/main.ts",
   output: {
-    file: "dist/main.js",
+    file: "main.js",

```

Comment:
I noticed that Obsidian wants this file in the root directory, otherwise it was having trouble finding the plugin on my side. If need be I can revert it out though.

Reply:
We probably have difference workflows, I don't build mine into the actual plugin folder. I build elsewhere and then drag main.js into the plugin folder when I'm ready to test changes. I don't think i'm super opinionated about where it builds though.